corehq/apps/sms/handlers/keyword.py

```diff
@@ -65,6 +68,48 @@ def handle_global_keywords(v, text, msg, text_words, open_sessions):
     return fcn(v, text, msg, text_words, open_sessions)
 
 
+def global_keyword_update(v, text, msg, text_words, open_sessions):
+
+    outbound_metadata = MessageMetadata(
+        workflow=WORKFLOW_KEYWORD,
+    )
+    if len(text_words) > 1:
+        keyword = text_words[1]
+        if keyword.upper() == LOCATION_KEYWORD:
+            site_code = text_words[2:]
+            if not site_code:
+                send_sms_to_verified_number(v, get_message(MSG_UPDATE_LOCATION_SITE_CODE, v),
+                                            metadata=outbound_metadata)
+                return True
+
+            site_code = site_code[0].lower()
+
+            location = Location.by_site_code(v.domain, site_code)
+            if location:
+                v.owner.set_location(location)
+                send_sms_to_verified_number(
+                    v,
+                    get_message(MSG_UPDATE_LOCATION_SITE_CODE_SUCCESS),
+                    metadata=outbound_metadata
+                )
+                return True
+            else:
+                send_sms_to_verified_number(
+                    v,
+                    get_message(MSG_UPDATE_LOCATION_SITE_CODE_NOT_FOUND, v).format(site_code),
```

Comment:
we shouldn't call `.format()` on the value returned from this, instead we should pass any variables as context to `get_message()`